dags.yaml

```diff
@@ -331,3 +331,14 @@ bqetl_desktop_platform:
       ]
     retries: 2
     retry_delay: 30m
+
+bqetl_cloud_monitoring_export:
+    schedule_interval: 0 * * * *
```

Comment:
I believe this is equivalent to `@daily`, and we should prefer that shorthand if it's what we use consistently elsewhere in this repo.

Reply:
This is hourly but yes `@hourly` would work